### ISSUE TITLE
Example is not loading

### ISSUE THREAD

**Issue by goran-insby** [tprebs/fluttermap_heatmap#15]
Flutter 3.16 
flutter_map: 6.0.0
flutter_map_heatmap: 0.0.6

example does not load heatmap, small adjustments to match flutter_map 6.0.0. were added, but that is not really substantial. There are no visible errors, data list is created with WeigthedLatLng, etc. 

![image](https://github.com/tprebs/fluttermap_heatmap/assets/45214405/60ddb20b-7426-40fe-985b-873417e26d23)

**Comment by translibrius** [comment]
Hey, just wanted to add that it is also not working for me on both Android and Web.

All I did was download the example project and run it. Using the default flutter_map version **^6.0.0**
When I try to use it in my personal project it also just does not paint anything. Not sure what more info I can provide.

![Screenshot_20231206_131510](https://github.com/tprebs/fluttermap_heatmap/assets/54362693/46ba48dd-5315-48b2-9b13-f97c0caab975)

**Comment by tprebs** [comment]
@goran-insby  I updated the sdk environment requirements  in the example. can you try again? The example project should require no changes to work. I just tried from a clean environment with no changes and it loads correctly. Do you have any more details to aid in debugging?

**Comment by tprebs** [comment]
@translibrius I have updated the sdk environment requirements in the example. can you try again? If you face the same issues, what version of dart and flutter are you using?

**Comment by translibrius** [comment]
@tprebs Hey sorry for the delay. I've updated the environment requirements like your commit, however the heatmap layer just doesn't show. I've logged the dataset and it's definitely there when it's passed to _InMemoryHeatMapDataSource_

My flutter & dart versions:
```
Flutter 3.16.0 • channel stable • https://github.com/flutter/flutter.git
Framework • revision db7ef5bf9f (4 weeks ago) • 2023-11-15 11:25:44 -0800
Engine • revision 74d16627b9
Tools • Dart 3.2.0 • DevTools 2.28.2
```

Let me know if I can help in some way.

**Comment by onetier41** [comment]
@tprebs hi, I am working on latest flutter version and everything is same as @translibrius, Heatmap is not visible

This is my environment

Flutter 3.16.3 • channel stable • https://github.com/flutter/flutter.git
Framework • revision b0366e0a3f (7 days ago) • 2023-12-05 19:46:39 -0800
Engine • revision 54a7145303
Tools • Dart 3.2.3 • DevTools 2.28.4

Thanks

**Comment by translibrius** [comment]
Im trying to debug the source code of the plugin, and seems like `ImageStreamCompleter load(HeatMapImage key, decode)` doesn't get called for some reason.

**Comment by translibrius** [comment]
I think I found the issue.
For people that this is also broken, please see my pull request and test if that fixes it for you as well. #16 

@onetier41 @goran-insby

**Comment by JEndler** [comment]
Ran into the same problem, #16 fixed it for me :)

**Comment by goran-insby** [comment]
@translibrius Yeah, now we are talking, it is a fix. Thanks a lot, works like a charm. I think this can be merged.

**Comment by tprebs** [comment]
Thanks @translibrius for the PR, this has been fixed and released in v0.0.7
